pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -195,12 +194,11 @@ func removeOldestJobs(cj *batchv1.CronJob, js []batchv1.Job, jc jobControlInterf
 		return
 	}
 
-	nameForLog := fmt.Sprintf("%s/%s", cj.Namespace, cj.Name)
-	klog.V(4).Infof("Cleaning up %d/%d jobs from %s", numToDelete, len(js), nameForLog)
+	klog.V(4).InfoS("Cleaning up jobs", "numToDelete", numToDelete, "jobCount", len(js), "cronjob", klog.KObj(cj))
 
 	sort.Sort(byJobStartTime(js))
 	for i := 0; i < numToDelete; i++ {
-		klog.V(4).Infof("Removing job %s from %s", js[i].Name, nameForLog)
+		klog.V(4).InfoS("Removing job", "cronjobName", js[i].Name, "cronjob", klog.KObj(cj))
```

Comment:
I have a question，if cronjob contains cronjobName? if so, let's only keep cronjob,if not, need modify the name?

Reply:
It's actually a batchjob so I updated it. Let me know if there's something else you prefer.